src/main/java/com/google/step/model/Deal.java

```diff
@@ -9,6 +9,8 @@
   public final String source;
   public final long posterId;
   public final long restaurantId;
+  public final String timestamp;
+  private double hotScore;
```

Comment:
this doesn't need to be an attribute, like how number of votes is not an attribute. It feels weird to suddenly have a private attribute here.

Reply:
but if its not an attribute, im not sure how to store the information? since it cant be a manager right